src/current/v23.2/live-migration-service.md

```diff
@@ -0,0 +1,612 @@
+- The *source of truth* is the database that serves reads and writes to the application during a live migration. A cutover switches the source of truth.
+- *Shadowing* is the replication of traffic on the target database. The LMS supports multiple [shadowing modes](#shadowing-modes).
+
+## Requirements
+
+- [Kubernetes](https://kubernetes.io/) cluster
+- [Helm](https://helm.sh/docs/intro/install/) package manager for Kubernetes
+
+#### Supported databases
+
+- [PostgreSQL]({% link {{ page.version.version }}/migrate-from-postgres.md %}) (source)
+- [MySQL]({% link {{ page.version.version }}/migrate-from-mysql.md %}) (source)
+- CockroachDB (source and target)
+
+## Installation
+
+To install the LMS, add the [Helm chart](https://molt.cockroachdb.com/lms/charts).
+
+When you install the `molt-lms` chart, the LMS proxy instances and orchestrator are initialized as Kubernetes pods:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+kubectl get pods
+~~~
+
+~~~
+NAME                                          READY   STATUS      RESTARTS      AGE
+lms-molt-lms-5cbd7c748-bzbgn                  1/1     Running     0             2m30s
+lms-molt-lms-5cbd7c748-g5zqf                  1/1     Running     0             2m34s
+lms-molt-lms-5cbd7c748-lbjjz                  1/1     Running     0             2m3
+lms-molt-lms-orchestrator-596d4b54d8-c4v97    1/1     Running     0             21m
+...
+~~~
+
+You will see `lms-molt-lms` pods that match the configured [number of LMS instances](#lms-instances), along with one `lms-molt-lms-orchestrator` pod.
+
+## Configuration
+
+To configure the LMS, override the values defined in the [Helm chart](#installation). For information on setting Helm chart values, see the [Helm documentation](https://helm.sh/docs/helm/helm_upgrade/).
+
+{{site.data.alerts.callout_info}}
+Values that are specified in both `lms` and `orchestrator` must have the identical setting.
+{{site.data.alerts.end}}
+
+This section describes the most important and commonly used values.
+
+#### Source dialect
+
+~~~ yaml
+lms:
+  sourceDialect: ""
+...
+orchestrator:
+  sourceDialect: ""
+~~~
+
+You **must** provide a string value for `sourceDialect`, which specifies the dialect of your source database. Supported dialects are:
+
+- `postgres`: PostgreSQL
+- `mysql`: MySQL
+- `cockroach`: CockroachDB
+
+#### Shadowing
+
+~~~ yaml
+lms:
+  shadowMode: none
+~~~
+
+`lms.shadowMode` specifies the shadowing behavior used by the LMS. This should depend on your specific migration requirements. For details, see [Shadowing modes](#shadowing-modes).
+
+#### LMS instances
+
+~~~ yaml
+lms:
+  replicaCount: 3
+~~~
+
+`lms.replicaCount` determines the number of LMS instances created as `lms-molt-lms` pods on the Kubernetes cluster, across which application traffic is distributed. This defaults to `3`.
```

Comment:
Similar to the previous comment, but `lms-molt-lms` may not be their pod name if they named the release something different or did a name override.